packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/EnumProvider.cs

```diff
@@ -36,10 +36,10 @@ protected EnumProvider(InputEnumType input)
 
         public CSharpType ValueType { get; }
         public bool IsExtensible { get; }
-        internal bool IsIntValueType { get; }
+        public bool IsIntValueType { get; }
```

Comment:
Can the consumer simply check the ValueType themselves vs having these extra public APIs?